packages/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -83,53 +85,47 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
     /// @param success        Whether the withdrawal transaction was successful.
     event WithdrawalFinalized(bytes32 indexed withdrawalHash, bool success);
 
-    /// @notice Emitted when the pause is triggered.
-    /// @param account Address of the account triggering the pause.
-    event Paused(address account);
-
-    /// @notice Emitted when the pause is lifted.
-    /// @param account Address of the account triggering the unpause.
-    event Unpaused(address account);
-
+    // TODO: move to a new inheritable contract so that this modifier can be reused in the
+    // xDM and bridge
```

Comment:
This coment is wrong. 
I do still need to expand the pause to the other bridge contracts, but these events don't need to be in them.